common/src/main/kotlin/ltd/redeye/vanguard/common/VanguardCore.kt

```diff
@@ -92,4 +91,8 @@ class VanguardCore(private val vanguardPlugin: VanguardPlugin) {
         return CompletableFuture.supplyAsync { this.storageDriver.loadPlayer(uuid) }
     }
 
+    fun getVanguardPlayer(name: String): CompletableFuture<VanguardPlayer?> {
```

Comment:
May be worth making this `CompletableFuture<Optional<VanguardPlayer>>` so we're not possible returning a null value in a future?